### ISSUE TITLE
Improve stdin character generation

### ISSUE THREAD

**Issue by Aaron1011** [Tiecoon/B7#34]
Currently, `StdinCharGenerator` can only generate characters within a specified range. It would be useful to be able to specify additional characters outside of that range (e.g. `\n`), to allow properly solving `tests/wyvern`

**Comment by Tiecoon** [comment]
The default constructor is set to the ascii range of 0x20-0x75. Which does include \n it's just a byproduct of the way it's being run that the binary isn't effected by the last character.

**Comment by Aaron1011** [comment]
Isn't `0x0a` the code for `\n`?

**Comment by Tiecoon** [comment]
It might be nice to be able to support multiple separate ranges in this case I guess

**Comment by Tiecoon** [comment]
Yeah ok I see what you mean, I don't think it will fix the test case output however. Just be careful anything with a null byte may effect strlen.

**Comment by twizmwazin** [comment]
The range implementation was to prevent null bytes causing short strings to be larger outliers than correct results. This could be replaced with a alternative constructor taking a Vec<u8> of all acceptable values.
